cdap-ui/app/cdap/components/Cloud/Profiles/ListView/ListView.scss

```diff
@@ -73,6 +73,45 @@ $row-height: 50px;
         &:last-child {
           border-bottom: 0;
         }
+
+        .profile-actions-popover {
+          .icon-cog-empty {
+            fill: white;
+            font-size: 23px;
```

Comment:
Is it just me or is the icon a little bigger than it should it be?

![screen shot 2018-04-23 at 11 02 45 am](https://user-images.githubusercontent.com/1452845/39144555-fb6255cc-46e5-11e8-8ddc-399fe110892b.png)

Also it would be nice if we use `rem` units instead of pixels. If I remember correctly the icon font sizes we use in other places to make it look enhanced is `1.3rem`. I might be wrong on the number. But we should square down on a standard number to be used consistently across UI.

Reply:
Talked with @ajainarayanan, we decided to keep the font-size as it is since it's according to design.